ci/build_container/Makefile

```diff
@@ -12,10 +12,11 @@ CXX ?= g++
 CXXFLAGS ?= -ggdb3 -fno-omit-frame-pointer -O2
 CFLAGS ?= -ggdb3 -fno-omit-frame-pointer -O2
 CPPFLAGS ?= -DNDEBUG
+BAZEL_WINDOWS_BUILD_TYPE ?= opt
```

Comment:
Can we set this in https://github.com/envoyproxy/envoy/blob/master/bazel/repositories.bzl#L88 based on the `config_setting` for Windows build? I.e. not have to specify it manually? Even better, I would name this `BAZEL_WINDOWS_DEBUG_BUILD` to make it clear, I think that's the key thing you want to extract from the `config_setting` here.